charts/barman/templates/pvc.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if or .Values.persistence.data.enabled .Values.persistence.recover.enabled }}
```

Comment:
Did you test if this `or` is required? What would happen if both are `false` and the template would solely consist of:
```yaml
---
```

Reply:
Nothing happens, it can handle it. The yaml-file isn't processed when both are `false` even without the `if or` statement. So I've removed it